### PR TITLE
Add AttendBox and SponsorBox components

### PR DESCRIPTION
**Pull request by warren**
# Description

This PR adds 2 new components, AttendBox and SponsorBox. They use the Header component made by Mike last weekend.

# Reviewing

Not much to review, just make sure the text renders. There should be a colorful header for each section and black subtext for each.

# Tests
I affirm that I've tested my code on:
- [x] Desktop
- [x] Tablet
- [x] Mobile
(Chrome tools counts)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,79 @@
...
+            />
+          </h2>
           <p>Tracks and workshop stuff here!</p>
         </div>
 
         <div>
-          <h2>Frequently Asked Questions</h2>
+          <h2>
+            <Header
+              contentProp="Frequently"
```
Comment (KenG98) [inline src/App.js]: typo (f -> faq)

Comment (warren) [reply src/App.js]: Good catch, fixed!

**Comment by lkwatson** [comment]
Please merge in master and get a code review from @maridematte or @sarah256

**Review by maridematte — APPROVED** [review]
Looking gud

**Review by lkwatson — APPROVED** [review]

**Pull request merged**